src/main/java/Duke.java

```diff
@@ -1,10 +1,227 @@
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Got it. I've added this task:");
+        System.out.println("\t\t" + "[" + taskList.get(index).getTaskSymbol() + "]" + "[" +
+                taskList.get(index).getStatusIcon() + "]" + " " + taskList.get(index));
+        System.out.println("\t Now you have " + taskIndex + " tasks in the list.");
+        System.out.println("\t____________________________________________________________");
+    }
+
+/*    private static void printFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(FILE_PATH);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            System.out.println(s.nextLine());
+        }
+    }*/
+
+    public static void loadProgress() throws FileNotFoundException {
+        //File contains something
+        String textInput;
+        Scanner s = new Scanner(f);
+        String[] taskArray = new String[4];
+        //Format of text: T | 1 | read book
+        while (s.hasNext()) {
+            textInput = s.nextLine();
+            taskArray = textInput.split(" \\| ");
+            if (taskArray[0].equalsIgnoreCase("T")) {
+                //Create Todo task
+                taskList.add(new Todo(taskArray[2]));
+                taskList.get(taskCount).setLoadStatus(taskArray[1]);
+                taskCount++;
+            }
+            if (taskArray[0].equalsIgnoreCase("D")) {
+                //Create Deadline Task
+                taskList.add(new Deadline(taskArray[2], taskArray[3]));
+                taskList.get(taskCount).setLoadStatus(taskArray[1]);
+                taskCount++;
+            }
+            if (taskArray[0].equalsIgnoreCase("E")) {
+                //Create Event Task
+                taskList.add(new Event(taskArray[2], taskArray[3]));
+                taskList.get(taskCount).setLoadStatus(taskArray[1]);
+                taskCount++;
+            }
+        }
+    }
+
+    private static void writeToFile(String filePath) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        Task a;
+        for (int i = 0; i < taskList.size(); i++) {
+            a = taskList.get(i);
+            if (a.getTaskSymbol().equals("T")) {
+                fw.write(a.getTaskSymbol() + " | " + a.getStatusNumber() + " | " + a.toString());
+                fw.write("\n");
+            }
+            if (a.getTaskSymbol().equals("D")) {
+                fw.write(a.getTaskSymbol() + " | " + a.getStatusNumber() + " | " + a.toString() + " | " +
+                        a.getDate());
+                fw.write("\n");
+            }
+            if (a.getTaskSymbol().equals("E")) {
+                fw.write(a.getTaskSymbol() + " | " + a.getStatusNumber() + " | " + a.toString() + " | " +
+                        a.getDate());
+                fw.write("\n");
+            }
+        }
+        fw.close();
     }
 }
```

Comment:
Suggestion: Duke java file is too long for a single file. You may probably reduce its length by using different classes to handle some methods.